src/pages/home/release-notes/index.md

```diff
@@ -7,6 +7,18 @@ Keywords:
 
 # Release notes
 
+## January 30, 2024
+
+### Android Assurance 2.2.1
+
+* Fixed an issue that caused large Assurance events to be sent with malformed payload.
+
+## January 29, 2024
+
+### Android Campaign Classic 2.1.2
+
+- Fix notification sticky behavior

```

Comment:
Can you change `-` to `*` for consistency? and also expand on how the behaviour was changed. 
You can update your GH release page with the fixes and we can trigger the workflow again. 

nit: @yangyansong-adbe We should handle converting `-` to `*` in our script.